enterprise/frontend/test/metabase-enterprise/formatting/whitelabel.cy.spec.js

```diff
@@ -122,21 +88,69 @@ describeWithToken("formatting > whitelabel", () => {
         .clear()
         .type(colors.additional4.hex);
       cy.findByText("Done").click();
+    });
+  });
 
-      changeThemeColor(33, colors.additional5.hex);
+  describe("company name", () => {
+    const COMPANY_NAME = "Test Co";
 
-      cy.get(".Icon-close").should("have.length", 10);
+    beforeEach(() => {
+      cy.log("**Change company name**");
+      cy.visit("/admin/settings/whitelabel");
+      cy.findByPlaceholderText("Metabase")
+        .clear()
+        .type(COMPANY_NAME);
+      // Helps scroll the page up in order to see "Saved" notification
+      cy.findByText("Application Name").click();
+      cy.findByText("Saved");
+      cy.findByDisplayValue(COMPANY_NAME);
+      cy.log("Company name has been updated!");
     });
 
-    it("should show color changes", () => {
+    it("changes should reflect in different parts of UI", () => {
+      cy.log("**--1. New company should show up on activity page--**");
+      cy.visit("/activity");
+      cy.findByText(`${COMPANY_NAME} is up and running.`);
+      cy.findByText("Metabase is up and running.").should("not.exist");
+
+      cy.log("**--2. New company should show up when logged out--**");
+      signOut();
+      cy.visit("/");
+      cy.findByText(`Sign in to ${COMPANY_NAME}`);
+
+      cy.log("**--3. New company should show up for a normal user--**");
+      signInAsNormalUser();
+      cy.visit("/activity");
+      cy.findByText(`${COMPANY_NAME} is up and running.`);
+      cy.findByText("Metabase is up and running.").should("not.exist");
+    });
+  });
+
+  describe("company color theme", () => {
+    beforeEach(() => {
+      cy.request("PUT", "/api/setting/application-colors", {
+        value: {
+          accent1: `#${colors.accent1.hex}`,
+          accent2: `#${colors.accent2.hex}`,
+          accent3: `#${colors.additional1.hex}`,
+          accent4: `#${colors.additional2.hex}`,
+          accent5: `#${colors.additional3.hex}`,
+          accent6: `#${colors.additional4.hex}`,
+          accent7: `#${colors.additional5.hex}`,
+          brand: `#${colors.primary.hex}`,
+          nav: `#${colors.nav.hex}`,
+        },
+      });
+    });
+
+    it("should reflect color changes", () => {
       signOut();
       cy.visit("/");
-      cy.contains("Sign in");
 
       // Note that if we have modified the logo, the entire background turns the brand color.
       // But if we _haven't_, as is the case now, then the existing logo is branded
       // As is the "Remember me" and "Sign in" inputs
-      cy.get(".Icon").should(
```

Comment:
This was dangerous to do because there are actually 2 icons on the login page. Tbh, not sure why it didn't fail before (probably because it checks only the first of the two).
The other icon is `.Icon-check` inside the checkbox for "Remember me".